site/docs/runtimes/deno.mdx

```diff
@@ -5,50 +5,17 @@ title: 'Running on Deno'
 
 # Running on Deno
 
-Kysely doesn't include drivers for Deno, but you can still use Kysely as a query builder or implement your own driver:
+It's possible to use SQLite with Deno by using [`kysely-deno-sqlite`](https://gitlab.com/soapbox-pub/kysely-deno-sqlite).
+This method uses [deno-sqlite](https://github.com/dyedgreen/deno-sqlite) under the hood.
```

Comment:
I don't think this is the right place to promote specific community dialects.
We do promote community dialects [here](https://github.com/kysely-org/kysely/blob/master/site/docs/dialects.md) and [here](https://github.com/kysely-org/awesome-kysely).

The original sentence is partially true, now that Deno supports `npm:<library>` installations and we've got a test proving `PostgresDialect` and `MysqlDialect` work in Deno (not SqliteDialect sadly). Should be re-worded to say that Kysely's PostgreSQL and MySQL dialects work in Deno, but for SQLite there are community dialects - and linking to the dialects page (https://github.com/kysely-org/kysely/blob/master/site/docs/dialects.md).